cds/modules/records/search.py

```diff
@@ -94,7 +92,7 @@ class KeywordSearch(RecordsSearch):
     class Meta:
         """Configuration for CERN search."""
 
-        index = schema_to_index(Keyword._schema)[0]
+        index = 'keywords-keyword-v1.0.0'
```

Comment:
@egabancho why this change?

Reply:
This is what breaking the app 😂 
The function `schema_to_index` requires application context since the introduction of index prefixes to Invenio-Search, and we where calling this functions, because it's defined at the class level, before the application was created.